.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 lib
 .DS_Store
 *.log
-.vscode
\ No newline at end of file
+.vscode
+yarn.lock
```

Comment:
So, to prevent commiting it every time unless it will be added, let's place it to .gitignore.